### PR TITLE
Add logging call for url `https://bios.app/v1/2015` #3211

### LINKED ISSUE #3211: Figure out where unknown URL access is coming from https://bios.app/v1/2015
I'm seeing this error in the console when running the app. Can you help to figure out where the URL is coming from? You might not see the same URL `https://bios.app/v1/2015` on your side, but I have seen it consistently. So if you know a good way to log it, I cna check it out on my side.

```
2021-09-24 11:16:41.051845+0800 AlphaWallet[32969:9032785] Task <D09B7024-CF2C-4776-9B82-37C20F358A85>.<27> finished with error [-1001] Error Domain=NSURLErrorDomain Code=-1001 "The request timed out." UserInfo={_kCFStreamErrorCodeKey=-2102, NSUnderlyingError=0x6000018178a0 {Error Domain=kCFErrorDomainCFNetwork Code=-1001 "(null)" UserInfo={_kCFStreamErrorCodeKey=-2102, _kCFStreamErrorDomainKey=4}}, _NSURLErrorFailingURLSessionTaskErrorKey=LocalDataTask <D09B7024-CF2C-4776-9B82-37C20F358A85>.<27>, _NSURLErrorRelatedURLSessionTaskErrorKey=(
    "LocalDataTask <D09B7024-CF2C-4776-9B82-37C20F358A85>.<27>"
), NSLocalizedDescription=The request timed out., NSErrorFailingURLStringKey=https://bios.app/v1/2015, NSErrorFailingURLKey=https://bios.app/v1/2015, _kCFStreamErrorDomainKey=4}
```

### PR DESCRIPTION
**Pull request by vladyslav-iosdev**
Closes #3211

### CODE REVIEW

File: AlphaWallet/Core/Coordinators/WalletBalance/PrivateBalanceFetcher.swift [hunk 1]
```diff
@@ -356,6 +357,7 @@ class PrivateBalanceFetcher: PrivateBalanceFetcherType {
         struct Error: Swift.Error {
         }
         let uri = originalUri.rewrittenIfIpfs
+        print("kkk: \(uri), address: \(address.eip55String), tokenId: \(tokenId)")
```
Comment (hboon) [inline AlphaWallet/Core/Coordinators/WalletBalance/PrivateBalanceFetcher.swift:360]: Oh, it's here!

**Review by hboon — COMMENTED** [review]
Wow. Thanks. Sorry you had to do this manually. But you figured it out! Let's leave this PR as it is for now. After we merge the logging PR, we can probably tweak this to do some logging when access to URLs fail.

**Review by hboon — COMMENTED** [review]
Ok. Now that logging is in, could you help to convert all these prints/NSLogs to `verbose()` with a short clue where/why they are being logged from and then merge it?

Eg.

```
verbose("Downloading token icon: \(url)…")
```

**Review by hboon — APPROVED** [review]
Looks good. But can you help split it into 2 commits? 1 for the logging change, another for the rest? :)

**Comment by hboon** [comment]
And replace this:

```
//NOTE: debug function that called suspicious request
if url.absoluteString.contains("https://bios.app/v1/2015") {
    verbose("Loading suspicious url: https://bios.app/v1/2015")
}
```

with:

```
verbose("Loading token icon URL: \(url.absoluteString)")
```

Let's just log all of them for now.

**Comment by hboon** [comment]
@vladyslav-iosdev should I review and merge?

**Comment by vladyslav-iosdev** [comment]
> @vladyslav-iosdev should I review and merge?

yep, it should be done

**Comment by hboon** [comment]
Rebase master and tweak commit message

**Review by hboon — APPROVED** [review]

**Pull request merged**